assets/app/view/game/entities.rb

```diff
@@ -22,6 +22,8 @@ def render
           players = players.rotate(i)
         end
 
+        bankrupt_players, players = players.partition(&:bankrupt)
```

Comment:
TIL the partition function 👍

Reply:
Since I don't know Ruby very well, I always have to search for solutions to most things. So follow my commits if you want to keep learning obscure random methods xD